MediaBrowser.Providers/Subtitles/SubtitleManager.cs

```diff
@@ -48,17 +47,18 @@ public class SubtitleManager : ISubtitleManager
             _localization = localizationManager;
         }
 
+        /// <inheritdoc />
+        public event EventHandler<SubtitleDownloadFailureEventArgs> SubtitleDownloadFailure;
+
+        /// <inheritdoc />
         public void AddParts(IEnumerable<ISubtitleProvider> subtitleProviders)
         {
             _subtitleProviders = subtitleProviders
-                .OrderBy(i =>
-                {
-                    var hasOrder = i as IHasOrder;
-                    return hasOrder == null ? 0 : hasOrder.Order;
-                })
+                .OrderBy(i => !(i is IHasOrder hasOrder) ? 0 : hasOrder.Order)
```

Comment:
```suggestion
                .OrderBy(i => (i is IHasOrder hasOrder) ? hasOrder.Order : 0)
```
IMHO it's easier to understand this way